f600233d1b154d507b782f84604bcb12/update/sys_script_include_5d76f38e970611106fa8b4b3f153afc4.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_g_dis_atat.Request</api_name>
+        <caller_access/>
+        <client_callable>false</client_callable>
+        <description>ATAT class that helps make general request to external resources (EDS, CSS, etc.).</description>
+        <name>Request</name>
+        <script><![CDATA[var Request = Class.create();
+Request.prototype = {
+    initialize: function() {
+		// methods
+		this.GET = "GET";
+		this.POST = "POST";
+		this.PATCH = "PATCH";
+		this.DELETE = "DELETE";
+		
+		// mutual auths
+		this.JKS = "jks443"; // CSS
+		this.MAUTH = "mauth"; // EDA/PDS
+    },
+    // Sends a request based on the request properties provided
+	// @para {object} - object that contains request properties endpoint, method, mutual auth
+	//  mutual auth property is optional
+	makeRequest: function (req) {
+		// Outbound REST message
+		var request = new sn_ws.RESTMessageV2();
+		
+		// Request set-up
+		request.setEndpoint(req.endpoint);
+		request.setHttpMethod(req.method);
+		request.setLogLevel("all");
+		if (req.mutualAuth) {
+			request.setMutualAuth(req.mutualAuth);	
+		}
+		
+		// Make request and return response
+		return request.execute();
+	},
+	// Checks the status code and adds an error property if request not successful
+	// @param {array} - array of accesstable success codes
+	// @param {response object} - response object returned from RESTMessageV2 execute() method
+	checkResponseCode: function (successCodes, response) {
+		if (!listContains(response.getStatusCode(), successCodes)) {
+			var errorMessage = "Status Code: " + response.getStatusCode() + ". ";
+			errorMessage += "Response: " + response.getBody() + ". ";
+			
+			return {
+				body: response.getBody(),
+				statusCode: response.getStatusCode(),
+				headers: response.getAllHeaders(),
+				error: errorMessage,
+			};
+		}
+
+		gs.info("External request successful.");
```

Comment:
Turn this down to debug and add in some identifying info (endpoint at least) so that it's more useful in the logs.

Reply:
Remove here and added it in `Request.getSuccessfulResponse`.